packages/aws-cdk/lib/api/util/sdk.ts

```diff
@@ -256,3 +276,16 @@ async function getCLICompatibleDefaultRegion(profile: string | undefined): Promi
 
     return region;
 }
+
+/**
+ * Find and return the configured HTTPS proxy address
+ */
+function httpsProxyAddress(): string | undefined {
+    if (process.env.http_proxy) {
```

Comment:
Didn’t know process.env is case insensitive

Reply:
Oops.